### PR TITLE
Add default keymap for Rubi Numpad

### PR DESCRIPTION
**Pull request by ohchiko**
## Description

As described in title.

### CODE REVIEW

**Review by noroadsleft — CHANGES_REQUESTED** [review]
The file path also needs to be `public/keymaps/r/rubi_default.json` – you have this in the wrong folder.

```
git mv public/keymaps/m/rubi_default.json public/keymaps/r/rubi_default.json
```

File: public/keymaps/m/rubi_default.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    "commit": "1a3064afb186328c17c679094b0f92cbd46844b5",
+    "layout": "LAYOUT",
+    "layers": [
+        [
+                                             "ENC_PRESS",
+            "KC_NLCK", "KC_PSLS", "KC_PAST", "KC_PMNS",
+            "KC_P7"  , "KC_P8"  , "KC_P9"  , "KC_PPLS",
+            "KC_P4"  , "KC_P5"  , "KC_P6"  ,
+            "KC_P1"  , "KC_P2"  , "KC_P3"  , "KC_PENT",
+                       "KC_P0"  , "LT(2, KC_PDOT)"
```
[the remark this answers is not in the document]
Comment (ohchiko) [reply public/keymaps/m/rubi_default.json]: Ah, I thought I put it in the right folder, my bad. Also, I never knew the `LT()` issue with whitespace, thanks for the correction.

[the remark this answers is not in the document]
Comment (ohchiko) [reply public/keymaps/m/rubi_default.json]: Resolved in https://github.com/qmk/qmk_configurator/pull/932/commits/2839dba82fdccabe73cab7170a2b4d483ae2aa7e.

**Review by noroadsleft — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**